site/testimonials/scrive.yaml

```
@@ -0,0 +1,7 @@
+---
+companyName: Scrive
+companyURL: https://scrive.com
+logoURL: logos/scrive.png
```

Maybe it needs this now?

```suggestion
logoURL: logos/Scrive.png
```